src/sentry/migrations/0931_disable_seer_automation_for_orgs.py

```diff
@@ -0,0 +1,280 @@
+
+    for org_id in TARGET_ORGANIZATION_IDS:
+        try:
+            org = Organization.objects.get(id=org_id)
+
+            _update_org_settings(org, OrganizationOption)
+
+            projects = Project.objects.filter(organization_id=org_id)
+            for project in projects:
+                _update_project_settings(project, ProjectOption)
+
+        except Organization.DoesNotExist:
+            continue
+
+
+def _update_org_settings(org: Organization, OrganizationOption: type[OrganizationOption]) -> None:
+    """Update organization-level seer automation settings."""
+    # Update sentry:default_seer_scanner_automation
+    scanner_key = "sentry:default_seer_scanner_automation"
+    try:
+        existing_scanner_option = OrganizationOption.objects.get(organization=org, key=scanner_key)
+        existing_scanner = existing_scanner_option.value
+    except OrganizationOption.DoesNotExist:
+        existing_scanner = None
+
+    if existing_scanner is False or existing_scanner is None:
+        OrganizationOption.objects.update_or_create(
+            organization=org, key=scanner_key, defaults={"value": False}
+        )
+
+    # Update sentry:default_autofix_automation_tuning
+    autofix_key = "sentry:default_autofix_automation_tuning"
+    try:
+        existing_autofix_option = OrganizationOption.objects.get(organization=org, key=autofix_key)
+        existing_autofix = existing_autofix_option.value
+    except OrganizationOption.DoesNotExist:
+        existing_autofix = None
+
+    if existing_autofix is False or existing_autofix is None:
+        OrganizationOption.objects.update_or_create(
+            organization=org, key=autofix_key, defaults={"value": "off"}
+        )
+
+
+def _update_project_settings(project: Project, ProjectOption: type[ProjectOption]) -> None:
+    """Update project-level seer automation settings."""
+    # Update sentry:seer_scanner_automation
+    scanner_key = "sentry:seer_scanner_automation"
+    try:
+        existing_scanner_option = ProjectOption.objects.get(project=project, key=scanner_key)
+        existing_scanner = existing_scanner_option.value
+    except ProjectOption.DoesNotExist:
+        existing_scanner = None
+
+    if existing_scanner is False or existing_scanner is None:
+        ProjectOption.objects.update_or_create(
+            project=project, key=scanner_key, defaults={"value": False}
+        )
+
+    # Update sentry:autofix_automation_tuning
+    autofix_key = "sentry:autofix_automation_tuning"
+    try:
+        existing_autofix_option = ProjectOption.objects.get(project=project, key=autofix_key)
+        existing_autofix = existing_autofix_option.value
+    except ProjectOption.DoesNotExist:
+        existing_autofix = None
+
+    if existing_autofix is False or existing_autofix is None:
+        ProjectOption.objects.update_or_create(
+            project=project, key=autofix_key, defaults={"value": "off"}
+        )
```

Comment:
Using `off` for one option and `False` for the other seems like a footgun for future us.